x-pack/plugins/security_solution/common/detection_engine/schemas/request/perform_bulk_action_schema.ts

```diff
@@ -6,13 +6,28 @@
  */
 
 import * as t from 'io-ts';
-import { bulkAction, queryOrUndefined } from '../common/schemas';
+import { BulkAction, queryOrUndefined, bulkActionUpdate } from '../common/schemas';
 
-export const performBulkActionSchema = t.exact(
-  t.type({
-    query: queryOrUndefined,
-    action: bulkAction,
-  })
-);
+export const performBulkActionSchema = t.union([
+  t.exact(
+    t.type({
+      query: queryOrUndefined,
```

Comment:
`query` is a common parameter for all the bulk actions, should it be lifted up in the schema?

Reply:
lifted up ^,  thanks for the suggestion